conans/client/loader_parse.py

```diff
@@ -29,7 +29,7 @@ class defining the Recipe, but also process possible existing generators
     """
     result = None
     for name, attr in conanfile_module.__dict__.items():
-        if "_" in name:
```

Comment:
Are you sure that dirty python things with "_" are only "_" and not containing "_"?